poglink/cogs/rates.py

```diff
@@ -142,6 +142,7 @@ async def on_ready(self):
                 last_rates = RatesStatus.from_dict(last_rates_dict)
 
                 # compare rates to last rates
+                #TODO: I think this is backwards; it should be last_rates.get_diff(rates)
```

Comment:
The diff would contain the same items, but the `old` vs `new` would be mixed up if this is indeed reversed.